integration/templates/ingress_controller_basic_values.go

```diff
@@ -10,14 +10,14 @@ controller:
   k8sAppLabel: nginx-ingress-controller
   metricsPort: 10254
 
-  replicas: 1
+  replicas: 3
```

Comment:
I think we still `1` replica as we are now forcing to schedule them on different nodes. As minikube only create one there will 2 pending if we use `3`. right @rossf7 ?

Reply:
Yes needs to be 1 for now. I hope with `kind` we can run multi node clusters but that's still a way off.